src/test/java/org/apache/sling/scripting/javascript/RepositoryScriptingTestBase.java

```diff
@@ -25,7 +25,6 @@
 import org.apache.sling.commons.testing.jcr.RepositoryTestBase;
 import org.apache.sling.scripting.javascript.internal.ScriptEngineHelper;
 
-
```

Comment:
Nit: please don't no whitespace only changes, it make it harder to track history of file changes.

Reply:
This still appears in the latest changeset.